7-toolchain/Dockerfile.fedora

```diff
@@ -1,24 +1,23 @@
-FROM fedora:27
-LABEL MAINTAINER Honza Horak <hhorak@redhat.com>
+FROM registry.fedoraproject.org/f27/s2i-core:latest
 
 ENV SUMMARY="Fedora variant of Red Hat Developer Toolset's C/C++ Toolchain from Software Collections" \
     DESCRIPTION="Platform for building C and C++ applications" \
-    NAME=perftools \
+    NAME=toolchain \
     VERSION=1 \
     RELEASE=1 \
```

Comment:
Maybe remove RELEASE now that is not (afaict) used?

Reply:
Fixed, thanks